### PR TITLE
Upload guide

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #996

### CODE REVIEW

File: docs/src/user/guide/upload.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+## Site details section
+
+For the site details you can create a new site or can select an existing site. To create a new site choose the 1️⃣ `Create New Site` option or to use the existing site choose the 2️⃣ `Use Existing Site` option.
+
+![site details](./img/upload-3.png)
+
+### Create New Site option
+
+By default the create new site option is selected. Fill the 1️⃣ required filds and upload the image of the site. To upload the image of the site click on the 2️⃣ `Upload site images` button.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
By default the create new site option is selected. Fill the 1️⃣ required fields and upload the image of the site. To upload the image of the site click on the 2️⃣ `Upload site images` button.
```

File: docs/src/user/guide/upload.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+![create new site](./img/upload-4.png)
+
+Upon clicking the popup will open, where you can select the images from the system. Click on the 1️⃣ `Browse` button to select the images from the system and then click on the 2️⃣ `upload chosen file` button to upload the images.
+
+![upload images](./img/upload-5.png)
+
+### Use Existing Site option
+
+Select this option by clickng on the 1️⃣ `Use Existing Site` radio button.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
Select this option by clicking on the 1️⃣ `Use Existing Site` radio button.
```

File: docs/src/user/guide/upload.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+![select on map](./img/upload-10.png)
+
+Upon selecting the site on the map latitude and longitude will be filled automatically.
+
+![select on map](./img/upload-11.png)
+
+### Type in coordinates option
+
+In this option you can enter the coordinates manually via two different types. You can either enter DMS coorrdinates or degree coordinate.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
In this option you can enter the coordinates manually via two different types. You can either enter DMS coordinates or degree coordinate.
```

File: docs/src/user/guide/upload.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+By default the degree coordinates (latitude and longitude) is selected.
+
+![degree coordinates](./img/upload-12.png)
+
+To enter the DMS coordinates click on the redio button available in front of the DMS.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
To enter the DMS coordinates click on the radio button available in front of the DMS.
```

File: docs/src/user/guide/upload.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+![DMS coordinates](./img/upload-13.png)
+
+
+## Observation details section
+
+Enter the collectors name and the notes in the 1️⃣ available fields and select the 2️⃣ date from the calander.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
Enter the collectors name and the notes in the 1️⃣ available fields and select the 2️⃣ date from the calendar.
```

File: docs/src/user/guide/upload.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+![observation ditails](./img/upload-14.png)
+
+To select the date from the calendar click on the calendar icon available in the date field.
+
+![calendar](./img/upload-15.png)
+
+## Measurements section
+
+Enter the measurment details in the 1️⃣ available fields. and the click on the
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
Enter the measurement details in the 1️⃣ available fields and the click on the next button
```

File: docs/src/user/guide/upload.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+To select the date from the calendar click on the calendar icon available in the date field.
+
+![calendar](./img/upload-15.png)
+
+## Measurements section
+
+Enter the measurment details in the 1️⃣ available fields. and the click on the 
+
+![measurment](./img/upload-16.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
![measurement](./img/upload-16.png)
```

File: docs/src/user/guide/upload.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+>Note: If the `Next` button is not clickable and the incomplete form message (Incomplete form. Please scroll up to complete the highlighted fields.) shown on the form that means you have forgotten to fill the required field.
+
+Fill all the required fields to proceed further. After filling all the required fields the incomplete form message will not show on the page and the `Next` button will be clickable. Click on the 1️⃣ next button for next step of data uploading.
+
+![measurment](./img/upload-17.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
![measurement](./img/upload-17.png)
```

File: docs/src/user/guide/upload.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+Upon clicking the next button the score form will open. Where you can select the group by clicking on the checkbox available in front of the group name and upload the images by clickin on the `Upload images` button available after the name of the group.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
Upon clicking the next button the score form will open. Where you can select the group by clicking on the checkbox available in front of the group name and upload the images by clicking on the `Upload images` button available after the name of the group.
```

File: docs/src/user/guide/upload.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+To access the `Add Record` form click on the 1️⃣ `Map` option available in the navigation bar.
+
+![map option](./img/upload-1.png)
+
+After clicking on this button, you will be redirected to the map page where you will find the 1️⃣ `Add Record` button, click on this button to open add record form.
+
+![add record button](./img/upload-2.png)
+
+Upon clicking on this button the add record form will open. This form have four sections
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
Upon clicking on this button the add record form will open. This form has four sections
```

File: docs/src/user/guide/upload.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+To enter the DMS coordinates click on the redio button available in front of the DMS.
+
+![DMS coordinates](./img/upload-13.png)
+
+
+## Observation details section
+
+Enter the collectors name and the notes in the 1️⃣ available fields and select the 2️⃣ date from the calander.
+
+![observation ditails](./img/upload-14.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
![observation details](./img/upload-14.png)
```

File: docs/src/user/guide/upload.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+![calendar](./img/upload-15.png)
+
+## Measurements section
+
+Enter the measurment details in the 1️⃣ available fields. and the click on the 
+
+![measurment](./img/upload-16.png)
+
+>Note: If the `Next` button is not clickable and the incomplete form message (Incomplete form. Please scroll up to complete the highlighted fields.) shown on the form that means you have forgotten to fill the required field.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
>Note: If the `Next` button is not clickable and the incomplete form message (Incomplete form. Please scroll up to complete the highlighted fields.) is shown on the form that means you have forgotten to fill the required fields.
```

File: docs/src/user/guide/upload.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+
+Fill all the required fields to proceed further. After filling all the required fields the incomplete form message will not show on the page and the `Next` button will be clickable. Click on the 1️⃣ next button for next step of data uploading.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/upload.md]:
```suggestion
Fill all of the required fields to proceed further. After filling all the required fields the incomplete form message will not show on the page and the `Next` button will be clickable. Click on the 1️⃣ next button for next step of data uploading.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by Jeremy-Prior** [comment]
@ketankartoza please address the merge conflicts when possible.

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 24 sec

**Pull request merged**